src/client/mod.rs

```diff
@@ -933,29 +933,11 @@ impl Client {
         #[cfg(feature = "voice")]
         self.voice_manager.lock().set_shard_count(shard_data[2]);
 
-        // This is kind of gross, but oh well.
-        //
-        // Both the framework and voice bridge need the user's ID, so we'll only
-        // retrieve it over REST if at least one of those are enabled.
-        #[cfg(any(all(feature = "standard_framework", feature = "framework"),
-                  feature = "voice"))]
+        #[cfg(feature = "voice")]
```

Comment:
I searched through the codebase if we still used `StandardFramework::id`, but other than its initialization in the `Framework` trait, there were none.